app/src/test/java/swati4star/createpdf/util/ImageUtilsTest.java

```diff
@@ -0,0 +1,17 @@
+package swati4star.createpdf.util;
+
+import com.itextpdf.text.Rectangle;
+
+import org.junit.Test;
+
+public class ImageUtilsTest {
+
+    @Test
+    public void testcalculateFitSize() {
+        float testWidth = 8.0f;
+        float testHeight = 12.0f;
+        Rectangle testDocumentSize = new Rectangle(5.0f, 5.0f, 5.0f, 5.0f);
```

Comment:
Add test case for 
- negative points
- invalid length (length < 0)
- rectangle is a point
- rectangle is a line